content/post/pga.md

```diff
@@ -0,0 +1,97 @@
+---
+author: vadim
+date: 2018-04-10
+title: "Public Git Archive"
+image: /post/pga/logo.png
+description: "We have released a large dataset for MLonCode - Public Git Archive (PGA). It contains 182,000 top-starred repositories on GitHub and takes 3 TB of disk space. The paper which describes it was accepted
+to Mining Software Repositories conference. This post tells the PGA's story."
+categories: ["science", "technical"]
+---
+
+[![logo](/post/pga/logo.svg#left)
+Dataset landing](https://github.com/src-d/datasets/PublicGitArchive)
```

Comment:
This link is wrong. It should be: https://github.com/src-d/datasets/tree/master/PublicGitArchive

Reply:
Thanks for noticing!